components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/no-unused-prop-types */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const Avatar = props => (
+  <span className={`avatar color-teal avatar--${props.size}`}>
+    <span className="avatar-user user-avatar" 
+    style={{
+      'backgroundImage': `url('${props.image}')`    
+    }} />
+    
+  </span>
+);
+
+Avatar.propTypes = {
+  size: PropTypes.oneOf(['xsmall', 'small', 'medium', 'large', 'xlarge', 'xxlarge']),
+  image: PropTypes.string
```

Comment:
I think that something is wrong with the linter, since on my computer this line is breaking due to the trailing comma rule. This isn't a critical issue, but something we should look into tomorrow.